bucket/networkmanager.json

```diff
@@ -1,13 +1,17 @@
 {
     "version": "2021.1.0",
     "description": "Tool for managing networks and network troubleshooting",
-    "homepage": "https://borntoberoot.net/NETworkManager/",
-    "license": "GPL-3.0",
+    "homepage": "https://github.com/BornToBeRoot/NETworkManager",
```

Comment:
Use previous homepage

Reply:
Ok fixed, I wrote version extraction regex to extract it from github release page, because github checkver method requires github homepage as you noted earlier. Though, I am not sure how flexible should such regexes be.